### PR TITLE
chore: support debug mode

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates-to: keymanapp/keyman.com#494
Depends-on: keymanapp/shared-sites#38

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ## START STANDARD SITE BUILD SCRIPT INCLUDE
 readonly THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 readonly BOOTSTRAP="$(dirname "$THIS_SCRIPT")/resources/bootstrap.inc.sh"
-readonly BOOTSTRAP_VERSION=v0.10
+readonly BOOTSTRAP_VERSION=chore/v0.11
```
Comment (mcdurdin) [inline build.sh:5]:
```suggestion
readonly BOOTSTRAP_VERSION=v0.11
```

**Review by darcywong00 — APPROVED** [review]
Lgtm

File: .htaccess [hunk 2]
```diff
@@ -1,9 +1,19 @@
+
+php_value display_errors off
+php_value display_startup_errors off
+
+<IfFile /var/www/html/_control/debug>
+  php_value display_startup_errors on
+  php_value display_errors stderr
+</IfFile>
+
```
Comment (darcywong00) [inline .htaccess:9]: Some of the related PRs had
```
php_value include_path ".:/var/www/html/_includes/"
```

Is that not needed or handled elsewhere?

Comment (mcdurdin) [reply .htaccess:9]: Currently, that is handled in /resources/keyman-site.conf. We should probably consolidate (even consider moving away from .htaccess for performance reasons) but that can happen in a later PR

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**